<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeIn</class>
 <widget class="QMainWindow" name="timeIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>31</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 52, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>651</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #303446;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #c6d0f5;
font-size: 30px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="playTime">
       <property name="styleSheet">
        <string notr="true">color: #c6d0f5;
font-size: 40px</string>
       </property>
       <property name="text">
        <string>PLAY TIME</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel">
       <property name="styleSheet">
        <string notr="true">color: #c6d0f5;
font-size: 40px</string>
       </property>
       <property name="text">
        <string>  00:00:00</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>1</y>
       <width>10</width>
       <height>580</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>211</width>
       <height>520</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,5,0,0,0,0,0,0,0">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="habit">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_2">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_3">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_4">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_5">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_6">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_7">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_8">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit_4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_9">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_10">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="habit_11">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #232634;
border-radius: 5%;</string>
        </property>
        <property name="text">
         <string>Habit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="statusOn_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>112</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>164</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>19</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>68</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="start_butt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10% ;
background-color: #232634;
color: #c6d0f5;</string>
        </property>
        <property name="text">
         <string>Start </string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="time_butt_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10% ;
background-color: #232634;
color: #c6d0f5;</string>
        </property>
        <property name="text">
         <string>Add time</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="timeLabel_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>115</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>530</y>
       <width>220</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>355</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>401</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusOn_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>497</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ea999c;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>⏸️</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>357</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>404</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_11">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>450</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>498</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 15px;
color: #a5adce;
font-family: Liberation Mono;
background-color: rgba(255, 255, 255, 0);
border-radius: 5%;</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #a5adce;
font-family: Liberation Mono;
background-color: #1e1e2e;
border-radius: 5%;</string>
    </property>
    <property name="text">
     <string>new activity</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
